src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -419,6 +419,8 @@ export default {
 		},
 		formatRepaymentCards(repayments) {
 			if (repayments.length === 0) return [];
+			const livesImpacted = Math.floor(this.userBalance / 25) + 1;
```

Comment:
It might be good to change this variable to `livesToImpact`. Using `livesImpacted` is really similar to the text on the page for the borrower carousel.